src/main/java/org/fungover/haze/Initialize.java

```diff
@@ -0,0 +1,35 @@
+package org.fungover.haze;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Initialize {
+
+	private static final Map<String, String> cliOptions = new HashMap<>();
+
+
+	public void importCliOptions(String[] args) {
+
+		for (int i = 0; i < args.length; i++) {
+			if (i + 1 == args.length)
+				break;
+			cliOptions.put(args[i], args[i + 1]);
+		}
+	}
+
+	public int setPort() {
```

Comment:
Since this is a method getting the preferred port number, shouldn't it be called getPort?
```suggestion
	public int getPort() {
```

Reply:
Your right! I was probably thinking im setting the server port :)